dpkg/scanner.go

```diff
@@ -239,6 +248,21 @@ Restart:
 		}
 		name := hdr.Get("Package")
 		v := hdr.Get("Version")
+
+		// Is this a valid package?
+		if name == "" || v == "" {
```

Comment:
So check if the bare-minimum is in the header, if it's not, either return a fatal or non-fatal error depending on if some Header keys are present? Is it possible to mention the significance of these Header keys in a comment?